CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+# dbt_hubspot_source v0.17.0
```

Comment:
Could you [bump the README version up](https://github.com/fivetran/dbt_hubspot_source/tree/bugfix/category_add?tab=readme-ov-file#step-2-install-the-package-skip-if-also-using-the-hubspot-transformation-package) to [">=0.17.0", "<0.18.0"]?

Reply:
thank you!